source/read-write-concern/read-write-concern.rst

```diff
@@ -12,8 +12,8 @@ Read and Write Concern
 :Status: Approved
 :Type: Standards
 :Server Versions: 2.4+
-:Last Modified: 2021-04-07
-:Version: 1.5.5
+:Last Modified: 2021-06-15
+:Version: 1.5.6
```

Comment:
This should be bumped to 1.6 as we're adding a new read concern mode.

Reply:
Done.